roles/sudoers/meta/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+galaxy_info:
+  author: Jim Enright (jenright@cloudera.com)
+  description: >
+    Adding specified users to FreeIPA sudoers group and creation of a sudo rule to allow passwordless sudo for the group. Existing users in the sudoers can be purged or retained depending on the value of the sudoers__purge_users_in_group flag.
```

Comment:
Format this line to within 120 chars.

Reply:
Should be fixed in https://github.com/cloudera-labs/cloudera.exe/pull/6/commits/40d424a3a31f6f3bf9b6d2314bc811de5c5b2a8c